src/client/content_cao.h

```diff
@@ -26,6 +26,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "itemgroup.h"
 #include "constants.h"
 #include <cassert>
+#include "server/serveractiveobject.h"
```

Comment:
Unnecessary include. This slows down the compile process.

```C++
class BoneOverride;
```

(a few lines below, though)